migrate/20230804_host_reboot.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    add_enum_value(:vm_display_state, "rebooting host")
```

Comment:
Customer might not have any idea what is "host"? But not have better name. Maybe just `rebooting`.

Reply:
I dig the brevity. Does it make sense that it looks the same for, say, a user...or administrative...initiated non-whole-host reboot? I kinda think it does.

Reply:
Just an idea came up my mind right now. It might be dummy.
VmHost one ➔ rebooting
Vm one ➔ restarting

Reply:
Given that the effect of a host reboot is just a very long VM reboot from a user's perspective, I think "rebooting" makes sense. I changed it.